unittests_bash/test_utils.bats

```diff
@@ -730,3 +730,50 @@ EOF
     echo "${output}"
     [[ "${EXPECTED_RESPONSE}" = "${output}" && "$status" -eq 1 ]]
 }
+
+@test "Get bundle arches: success" {
+    RENDER_OUT_BUNDLE=$(cat <<EOF
+{
+    "schema": "olm.bundle",
```

Comment:
I don't see the archs label in this render_out. I wonder how the test passes

Reply:
The label "operatorframework.io/arch" is under `properties.value.data`, if we decode a 64-encoded string then the CSV data will be there.

Reply:
Ah, I see. thanks!

Reply:
Doesn't this change for the version of `opm` or the parameters passed to rendering? Recent versions of `opm` should be able to generate it without base64 encoding everything.

Reply:
My version is pretty recent `Version: version.Version{OpmVersion:"649dfc4dc", GitCommit:"649dfc4dc0a7fca1c25c7678665a55ad8ce53040", BuildDate:"2024-11-20T17:46:14Z", GoOs:"linux", GoArch:"amd64"}`. Let me check whether I can pass some parameters to render without base64 encoding

Reply:
I tested with opm version 1.40.0 that is used in konflux-test image and indeed it renders bundleImage without the base64 encoding. I updated this PR to reflect that.